tests/integration/bgp.multihop/01-global.yml

```diff
@@ -29,6 +29,10 @@ bgp.multihop.sessions:
 - dut:
   pe2:
     passive: True
+- dut:
```

Comment:
This makes bgp.multihop functionality dependent on the bgp.local_as functionality. Remove from the PR.

Reply:
moved to separate test case in different PR